src/zcl_abapgit_objects.clas.abap

```diff
@@ -1083,8 +1084,12 @@ CLASS ZCL_ABAPGIT_OBJECTS IMPLEMENTATION.
     li_obj = create_object( is_item     = rs_files_and_item-item
                             iv_language = iv_language ).
     li_obj->mo_files = lo_files.
+
     CREATE OBJECT lo_xml.
-    li_obj->serialize( lo_xml ).
+    CREATE OBJECT lo_log.
+
+    li_obj->serialize( io_xml      = lo_xml
+                       ii_log      = lo_log ). "to be replaced by item logger
```

Comment:
also a change to central logic, `lo_log` will be garbage collected, so any potential errors will not be shown to the user?